client/src/js/references/components/__tests__/SelectItems.test.js

```diff
@@ -0,0 +1,18 @@
+import { ReferenceSelectItem } from "../ReferenceSelectItem";
+
+describe("<ReferenceSelectItem>", () => {
+    const props = {
+        reference: {
+            name: "foo",
+            user: {
+                id: "bar"
+            },
+            otu_count: "baz",
+            created_at: "bee"
+        }
+    };
+    it("should render", () => {
+        const wrapper = shallow(<ReferenceSelectItem {...props} />);
+        expect().toMatchSnapshot();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No arguments were passed to expect(). (jest/valid-expect)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4644906)